lib/declare_schema/extensions/active_record/fields_declaration.rb

```diff
@@ -15,7 +15,7 @@ def fields(table_options = {}, &block)
       @table_options        = table_options
 
       if block
-        dsl = DeclareSchema::FieldDeclarationDsl.new(self, null: false)
+        dsl = DeclareSchema::FieldDeclarationDsl.new(self)
```

Comment:
I'd forgotten that previously this could be overridden on a per-model basis. Oh well, we never found that helpful.